2.6/root/usr/share/container-scripts/mongodb/replset-functions.sh

```diff
@@ -0,0 +1,97 @@
+# This file contains functions	 for replSet manipulation
+
+# cache_container_addr waits till the container gets the external IP address and
+# cache it to disk
+function cache_container_addr() {
+  echo -n "=> Waiting for container IP address ..."
+  local i
+  for i in $(seq ${MAX_ATTEMPTS}); do
+    if ip -oneline -4 addr show up scope global | grep -Eo '[0-9]{,3}(\.[0-9]{,3}){3}' > "${HOME}"/.address; then
```

Comment:
Is it ok that in file could be more than one IP address?

Output of this command on Vagrant:
```
10.0.2.15
10.0.2.255
172.17.0.1
```
Output of this command on my host:
```
10.34.129.229
10.34.129.255
10.200.136.146
10.200.139.255
192.168.122.1
192.168.122.255
```